### PR TITLE
Add why kwok to ReadME

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?

/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: site/content/en/_index.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
 - `kwok` is the cornerstone of this project, responsible for simulating the lifecycle of fake nodes, pods, and other Kubernetes API resources.
 - `kwokctl` is a CLI tool designed to streamline the creation and management of clusters, with nodes simulated by `kwok`.
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
+Currently, KWOK can maintain 1k nodes and 100k pods per laptop.
+- Fast: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
+Currently, KWOK can create 20 nodes or 20 pods per second.
```
Comment (wzshiming) [inline site/content/en/_index.md:28]:
```suggestion
- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
Currently, KWOK can reliably maintain 1k nodes and 100k pods.
- Fast: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
Currently, KWOK can create 20 resources (nodes or pods) per second.
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ Please see [our website] for more in-depth information.
 
 <img width="700px" src="./demo/manage-clusters.svg">
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
+Currently, KWOK can maintain 1k nodes and 100k pods per laptop.
+- Fast: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
+Currently, KWOK can create 20 nodes or 20 pods per second.
```
Comment (wzshiming) [inline README.md:27]:
```suggestion
- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
Currently, KWOK can reliably maintain 1k nodes and 100k pods.
- Fast: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
Currently, KWOK can create 20 resources (nodes or pods) per second.
```

File: README.md [hunk 3]
```diff
@@ -19,6 +19,16 @@ Please see [our website] for more in-depth information.
...
+- Compatibility: KWOK works with any tools or clients that are compliant with Kubernetes APIs, such as kubectl, helm, kui, etc.
+- Portability: KWOK has no specific hardware or software requirements. You can run it using pre-built images, once Docker or Nerdctl is installed. Alternatively, binaries are also available for all platforms and can be easily installed.
```
Comment (windsonsea) [inline README.md:29]:
```suggestion
- Portability: KWOK has no specific hardware or software requirements. You can [run it using pre-built images], once Docker or Nerdctl is installed. Alternatively, binaries are also available for all platforms and can be easily installed.
```
[run it using pre-built images]: {{< relref "/docs/user/all-in-one-image" >}}

File: README.md [hunk 4]
```diff
@@ -19,6 +19,16 @@ Please see [our website] for more in-depth information.
 
 <img width="700px" src="./demo/manage-clusters.svg">
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
```
Comment (windsonsea) [inline README.md:24]:
```suggestion
- Lightweight: You can [simulate thousands of nodes] on your laptop without significant consumption of CPU or memory resources.
```
[simulate thousands of nodes]: {{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}

File: site/content/en/_index.md [hunk 5]
```diff
@@ -20,6 +20,16 @@ KWOK stands for Kubernetes WithOut Kubelet. So far, it provides two tools:
 - `kwok` is the cornerstone of this project, responsible for simulating the lifecycle of fake nodes, pods, and other Kubernetes API resources.
 - `kwokctl` is a CLI tool designed to streamline the creation and management of clusters, with nodes simulated by `kwok`.
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
```
Comment (windsonsea) [inline site/content/en/_index.md:25]:
```suggestion
- Lightweight: You can [simulate thousands of nodes] on your laptop without significant consumption of CPU or memory resources.
```
[simulate thousands of nodes]: {{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}

**Review by windsonsea — COMMENTED** [review]
Whether we could provide some links if possible, to attract readers to have a try immediately.

File: README.md (hunk 4 above)
Comment (kerthcet) [reply README.md:24]: I have no idea whether this will work, but updated as advised. And I'll take a review then.

Comment (kerthcet) [reply README.md:24]: Actually this doesn't work. see https://deploy-preview-451--k8s-kwok.netlify.app/

Comment (kerthcet) [reply README.md:24]: Fixed

**Pull request merged**